tools/flamebench

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Could this script be written in Deno instead of bash?

Reply:
Done